packages/viewer/src/lib/transforms/transformation-delta.ts

```diff
@@ -1,26 +1,23 @@
 import { Matrix4, Quaternion, Vector3 } from '@vertexvis/geometry';
 
 export const ALMOST_ONE = 0.9999;
+
 /**
- * For any single vector, there are an infinite number of potential orthogonal vectors. This function will determine
- * one orthogonal vector by crossing the provided vector with a unit vector in the positive X, Y, or Z directions.
+ * Calculates orthogonal coordinate frame with z-Vector pointing towards N
```

Comment:
Minor nit: could we update this comment to be a bit more clear? I'm not sure I fully follow what the `z-Vector` or `coordinate frame` are in the context of the SDK